engine/bsql_engine/io/cio.pxd

```diff
@@ -163,7 +163,7 @@ cdef extern from "../include/engine/engine.h":
         TableScanInfo getTableScanInfo(string logicalPlan)
 
 cdef extern from "../include/engine/initialize.h":
-    cdef void initialize(int ralId, int gpuId, string network_iface_name, string ralHost, int ralCommunicationPort, bool singleNode, map[string,string] config_options) except +raiseInitializeError
+    cdef void initialize(int ralId, int gpuId, string network_iface_name, string ralHost, int ralCommunicationPort, bool singleNode, map[string,string] config_options, float host_memory_quota) except +raiseInitializeError
```

Comment:
why are you sending `host_memory_quota` as a separate parameter when you could pass it in with all the other config_options?

Reply:
I put this before considering the host threshold to be a config parameter, removed.